SocialCareCaseViewerApi/V1/Controllers/FormSubmissionController.cs

```diff
@@ -78,11 +78,6 @@ public IActionResult CreateSubmission([FromBody] CreateCaseSubmissionRequest req
             {
                 var createdSubmission = _formSubmissionsUseCase.ExecutePost(request).Item1;
 
-                if (createdSubmission.SubmissionId == null)
-                {
-                    return StatusCode(500, "Case submission created with a null submission ID");
-                }
-
```

Comment:
Deleted because SubmissionId could never be null, the if conditional could never be true